nipype/info.py

```diff
@@ -136,6 +137,7 @@ def get_nipype_gitversion():
     'nibabel>=%s' % NIBABEL_MIN_VERSION,
     'networkx>=%s' % NETWORKX_MIN_VERSION,
     'numpy>=%s' % NUMPY_MIN_VERSION,
+    'numpy>=%s ; python_version >= "3.7"' % NUMPY_MIN_VERSION_37,
```

Comment:
@satra @oesteban This is intended to address https://github.com/nipy/nipype/pull/2775#issuecomment-438312643. I've tested locally with pip installs of the `bdist` wheel in different conda environments. Added a test to CI, so we'll see how that goes.